pkg/manifests/manifests.go

```diff
@@ -43,180 +46,186 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/klog/v2"
 	apiregistrationv1 "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1"
 )
 
+type Assets struct {
+	Mutex sync.RWMutex
```

Comment:
(nit) a regular Mutex is probably enough

```suggestion
	Mutex sync.Mutex
```